ecf/defs/prod00.def

```diff
@@ -1,6 +1,6 @@
 extern /prod18/gdas/jgdas_forecast
 extern /prod18/gdas/atmos/post
-extern /prod18/gdas/enkf/post
+extern /prod18/enkfgdas/post
```

Comment:
These are new changes added in these `.def` files that were agreed to be added in a follow-up PR in the meeting this AM.  This PR is already big to add any additional changes.